kickstarter/android-oss
# NTV-570: Firebase Crashes

# 📲 What

- Two spiking crashes happening lately, those two are "old friends" happening since versions 1.9.0 and 2.18.1 not able to reproduce them locally, and the long term solution will come after some architecture changes down the road, so this is an attempt to reduce/eliminate them with "workarounds"

# 🤔 Why
- [Firebase crash report 1](https://console.firebase.google.com/u/1/project/android-external-release/crashlytics/app/android:com.kickstarter.kickstarter/issues/c424493a75af12b73b047368f6acf16d?time=last-seven-days&versions=3.3.0%20(2013150891)&sessionEventKey=62C71465014900017A3AAC22ACE86773_1696129384934804493)

- [Firebase crash report 2](https://console.firebase.google.com/u/1/project/android-external-release/crashlytics/app/android:com.kickstarter.kickstarter/issues/5b7e4ace6007d59fcd90e428?time=last-seven-days&versions=3.3.0%20(2013150891)&sessionEventKey=62C71565017F000171C34400BF97DE7A_1696128986358397836)

|  |  |

# 📋 QA

- Load the Discovery screen, make sure you can log in, log out
- Load the screen discovery, make sure the "since your where gone card" loads correctly
- Load projects on discovery on different tabs, scroll as much as you want
- Load addOns screen in several projects
- Try to change you pledge (no need to actually change it, just load the pledge screen using that flow)

# Story 📖

[NTV-570](https://kickstarter.atlassian.net/browse/NTV-570)
